proto/tss/v1beta1/query.proto

```diff
@@ -147,3 +147,11 @@ message AssignableKeyRequest {
 }
 
 message AssignableKeyResponse { bool assignable = 1; }
+
+message ValidatorKeyRequest { string address = 1; }
+
+message ValidatorKeyResponse {
+  message Keys { repeated bytes keys = 1; };
```

Comment:
is this necessary?

Reply:
protobuf doesn't allow `repeated` in the map value type